sos/report/plugins/__init__.py

```diff
@@ -1449,11 +1449,16 @@ def time_filter(path):
                     continue
 
                 try:
-                    filestat = os.stat(_file)
+                    file_size = os.stat(_file)[stat.ST_SIZE]
                 except OSError:
                     self._log_info("failed to stat '%s'" % _file)
-                    continue
-                current_size += filestat[stat.ST_SIZE]
+                    # if _file is a broken symlink, we should collect it,
+                    # otherwise continue
+                    if os.path.islink(_file):
+                        file_size = 0
```

Comment:
I think we should only log a failure if it is not a broken symlink in this case, no?

The logged message would make me think "ok, we didn't collect that", but then seeing that (broken) collection in the archive would make me stop for a moment.

Reply:
Ah, good point, will fix it.